### PR TITLE
refactor(video-conferencing-service): fix lint issues and run prettier

### PR DESCRIPTION
**Pull request by anitsingh-sf**
SFO-81

## Description

refactor(video-conferencing-service): fix lint issues and run prettier

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/video-conferencing-service/src/controllers/video-chat-session.controller.ts [hunk 1]
```diff
@@ -235,20 +238,25 @@ export class VideoChatSessionController {
...
+      await this.auditLogRepository.create(auditLogPayload);
       throw new HttpErrors.BadRequest(errorMessage);
     }
 
-    return this.videoChatSessionRepository.updateById(videoSessionDetail.id, {
+    await this.videoChatSessionRepository.updateById(videoSessionDetail.id, {
       endTime: new Date(),
     });
+
+    auditLogPayload.after = 'Successful End session';
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:256]:
```suggestion
    auditLogPayload.after = { response:  'Successful End session' };
```

Comment (anitsingh-sf) [reply services/video-conferencing-service/src/controllers/video-chat-session.controller.ts]: done

**Comment by samarpan-b** [comment]
fix sonar

**Comment by anitsingh-sf** [comment]
> fix sonar

done

**Pull request merged**